test_communication/CMakeLists.txt

```diff
@@ -127,6 +127,32 @@ if(BUILD_TESTING)
     )
   endfunction()
 
+  function(custom_security_test_c target)
+    set(SKIP_TEST "")
+    ament_add_gtest(
+      "${target}${target_suffix}" ${ARGN}
+      TIMEOUT 90
+      ${SKIP_TEST}
+      APPEND_LIBRARY_DIRS "${append_library_dirs}"
+      ENV
+      RCL_ASSERT_RMW_ID_MATCHES=${rmw_implementation}
+      RMW_IMPLEMENTATION=${rmw_implementation}
+      ROS_SECURE_ROOT="${VALID_ROS_SECURE_ROOT}"
+    )
+    target_link_libraries(${target}${target_suffix}
```

Comment:
Thanks for the pointer! fixed in https://github.com/ros2/system_tests/pull/204/commits/d26f8c1c63234316fa7374bb040de3af69d93d5d